framework/src/st_things/things_stack/fota/fmwup_util_http.h

```diff
@@ -0,0 +1,27 @@
+/* ****************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include <tinyara/config.h>
+#include <protocols/webserver/http_keyvalue_list.h>
+
+#ifndef FMWUP_UTIL_HTTP_H_
```

Comment:
1. exchange locations of conditional and including header
```
#ifndef
#define

#include
```

2. Why do you include those headers in here?>